### ISSUE TITLE
Strange problem with Admintools

### ISSUE THREAD

**Issue by Zamarza** [carcam/plg_readingtime#20]
Hello,

I am facing a very strange problem with the plugin  Reading Time and Admin Tools.

When I am logged in as administrator to my back end, if I open my website in another tab, I am automatically logged out from my back end by Admin Tools.

I contacted the support and they say that you are doing something which is not allowed by a front end application. If I disable your plugin I am not logged out. They say it is a security question.

Please would you tell me if you are aware of that problem because it is annoying?

Thanks for your reply
Amar Guillen

**Comment by carcam** [comment]
Hi Amar,
that's indeed annoying!! I'm not aware of such problem. Will investigate it too. If you have more information I can check, please let me know.

Thanks for reporting it!!!

**Comment by Zamarza** [comment]
I do not have more information.
If you want to investigate thru a test website, let me know.
I will send you the link.

Sincerely
Amar Guillen

**Comment by carcam** [comment]
Hi Amar,
can you please provide credentials to check the admin tools issue and let me know how to reproduce the issue ;)

Sned them to carlos@hepta.es

**Comment by Zamarza** [comment]
Hello Carlos,

 

Here are the credentials for my test website:

 

*	Website:  <https://dev.guillenphoto.com> https://dev.guillenphoto.com
*	Admin: https://dev.guillenphoto.com/administrator/index.php?plano3512
*	id: tempUqaTAsiHWAKc
*	password: BcR07Gf55X19h38acHjQyEMH4CbTnT55

 

If you want to see the plugin Reading time in action: https://dev.guillenphoto.com/en/why-and-how-defining-a-photograph.html

 

There is no cache. JCH is disabled.

 

Thanks for your help
Waiting for some news 

Sincerely
Amar GUillen

 

 <https://www.guillenphoto.com/>  

Amar Guillen: The Passion for the Wonders of Nature.

 

Reconnect to Nature with my Fine Art Prints.

 

Website:  <https://www.guillenphoto.com/> https://www.guillenphoto.com

 

Phone: (+1) 214-769-6825
Email:  <mailto:contact@guillenphoto.com> contact@guillenphoto.com

Skype : guillenphoto

 

 <https://www.instagram.com/amarguillen/>    <https://www.facebook.com/amarguillen>    <https://www.linkedin.com/in/amarguillen>

**Comment by carcam** [comment]
Hi Amar,
thank you very much for details. I have been able to see the issue as I think it's related to K2 and how I process data there. Can you point me to the ticket on Akeeba forum so that I can ask them where is the exact problem?

Thanks!!

**Comment by Zamarza** [comment]
Hello,

 

Thank you for your time.

 

Here is the ticket.

https://www.akeebabackup.com/support/admin-tools/Ticket/33324:how-can-i-allow-a-plugin-to-an-admin-query-string.html

 

The ticket is closed.

I hope you will find a way to solve the problem.

 

Sincerely

Amar

 

 <https://www.guillenphoto.com/>  

Amar Guillen: The Passion for the Wonders of Nature.

 

Reconnect to Nature with my Fine Art Prints.

 

Website: https://www.guillenphoto.com <https://www.guillenphoto.com/> 

 

Phone: (+1) 214-769-6825
Email:  <mailto:contact@guillenphoto.com> contact@guillenphoto.com

Skype : guillenphoto

 

 <https://www.instagram.com/amarguillen/>    <https://www.facebook.com/amarguillen>    <https://www.linkedin.com/in/amarguillen>

**Comment by carcam** [comment]
Hi Amar,
thank you very much for all the information and your cooperation with this issue.

I am afraid this is not an issue related to Estimated Reading Time. I have stablished this procedure to reproduce the issue:

1. Login to backend of the site
2. Go to K2 component administration
3. Go to K2 items
4. Open your site in a new window
5. Go to Blog
6. Now your backend session is closed

With this procedure I have replicated the issue with Estimated Reading Time plugin enabled and disabled and in both cases, the issue persisted.

Also this plugin do not make any network call to backend and you can easily check that if you enable the plugin and see the network tab of your browser inspector tools, there is no request from this plugin and none of the requests seem to be to the administrator.

You have indeed several request to files in non-standard parts of your site like: /data plugins/.../plugin... and other parts and I wonder if these are the issues you are facing. If so, maybe enabling JCH js and css combination can help to solve this issue as it will prevent requesting the files from those folders.

In any case as I stated, it's not related to this plugin.

Best!!

**Comment by Zamarza** [comment]
Hello Carlos,

thank you very much for your time.

You are right. It is not related to the plugin. I have uninstalled it and the problem still occurs.
You know that it was not the case before installing it. It guess it is a bad conjunctions of facts.
So I keep your plugin in my website because it works well.

I found a way to work.
I open the administration in one browser.

I open my website in another one. And it works. I am not logged out.

So thanks for all
All my best
Amar
